test/unit/specs/util/loanUtils.spec.js

```diff
@@ -154,4 +154,40 @@ describe('loanUtils.js', () => {
 			expect(result).toEqual(['Agriculture']);
 		});
 	});
+	describe('DropdownPrices', () => {
+		const unreservedAmount = 2000;
+		const minAmount = 25;
+		const enableFiveDollarsNotes = false;
+
+		it('should return 25 as first option', () => {
+			const result = getDropdownPriceArray(unreservedAmount, minAmount, enableFiveDollarsNotes);
+			expect(result[0]).toEqual('25');
+		});
+
+		it('should return 500 as last option if amount left is greater than 500', () => {
+			const result = getDropdownPriceArray(unreservedAmount, minAmount, enableFiveDollarsNotes);
+			const lastOption = result.length - 1;
+			expect(result[lastOption]).toEqual('500');
+		});
+
+		it('should return amount left or closer $25 increment as last option if amount left is less than 500', () => {
+			const unreservedAmountLess500 = 325;
+			const result = getDropdownPriceArray(unreservedAmountLess500, minAmount, enableFiveDollarsNotes);
+			const lastOption = result.length - 1;
+			expect(result[lastOption]).toEqual('325');
+		});
+
+		it('should return 5 as first option if $5 notes test enabled', () => {
+			const FiveDollarsNotes = true;
```

Comment:
We usually use naming like `fiveDollarsNotes` or `FIVE_DOLLAR_NOTES` for constants. Or in this case, `true` can just be passed into `getDropdownPriceArray` directly without this variable.